### ISSUE TITLE
Crash on Windows 7 32bit

### ISSUE THREAD

**Issue by Halfshadow** [m64p/mupen64plus-GLideN64#124]
Hello, I want to use your emulator with Cruis'n Exotica, I heard that GLideN64 fixed the emulation, but your emulator crashes when I try to load any rom in Windows 7 32bit... This yesterday and also this morning with your new updated version.

What are the minimum requirements with your emulator? New framework versions etc. Or you mistaken something in the 32 bit version?

**Comment by loganmc10** [comment]
You mean you have the 32 bit version of Windows installed correct? If so, I can't really help, I don't have a system like that to test on.

**Comment by Halfshadow** [comment]
Yep, I have the 32 bit version. I tired also the 64 bit in my friend pc, but however the emulation with Exotica is very slow. And it's computer have 4GB ram.
